server/channels/db/migrations/mysql/000110_create_desktop_tokens.up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS DesktopTokens (
+    DesktopToken varchar(64) NOT NULL,
+    ServerToken varchar(64) NULL,
+    UserId varchar(26) NULL,
+    CreateAt bigint NOT NULL,
+    PRIMARY KEY (DesktopToken)
```

Comment:
The main fetch SQL query seems to use 

```
Where{sq.Eq{"DesktopToken": desktopToken},
sq.Eq{"ServerToken": serverToken},
sq.GtOrEq{"CreateAt": minCreateAt},
}
```

Do we want to have a compound index for that? Probably worth looking at which would be the most common queries and see if we want to add an index. Now would be the right time to do it, otherwise, it takes longer to create an index later.

Reply:
Yes, that's a good idea. We originally had an index on just the PK and createat which was enough, I'll change it to a compound of the 3.